### PR TITLE
ztp: ptp add ptpconfig for High Availability

### PR DESCRIPTION
**Pull request by aneeshkp**
4.16 : ptp operator enables HA via ptpconfig.
This PR adds config required to enable HA

### CODE REVIEW

**Comment by hhassid** [comment]
lgtm

File: ztp/source-crs/PtpConfigForHA.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: ptp.openshift.io/v1
```
Comment (sabbir-47) [inline ztp/source-crs/PtpConfigForHA.yaml:1]: @aneeshkp There are other configs available in the source-crs for HA. Would you mind adding a comment at the beginning on when and why user needs to use it?

**Review by nishant-parekh — APPROVED** [review]
/lgtm

**Pull request merged**